src/SonarScanner.MSBuild.Tasks/WriteProjectInfoFile.cs

```diff
@@ -106,7 +106,7 @@ public override bool Execute()
         }
         else
         {
-            Log.LogWarning(Resources.WPIF_MissingOrInvalidProjectGuid, FullProjectPath);
+            Log.LogMessage(MessageImportance.High, Resources.WPIF_MissingOrInvalidProjectGuid, FullProjectPath);
```

Comment:
We want to set high importance so the message will be displayed even when `dotnet build` is called without a verbosity parameter.

The logs will look like this:
```
  Determining projects to restore...
  All projects are up-to-date for restore.
  WorkerExtensions -> D:\src\playgroud\AzureFunctions\AzureFunctions\obj\Debug\net8.0\WorkerExtensions\buildout\Microsoft.Azure.Functions.Worker.Extensions.dll
  Sonar: The project does not have a valid ProjectGuid. Analysis results for this project will not be uploaded. Project file: D:\src\playgroud\AzureFunctions\AzureFunctions\obj\Debug\net8.0\WorkerExtensions\WorkerExtensions.csproj
  Sonar: (WorkerExtensions.csproj) Project processed successfully
  AzureFunctions -> D:\src\playgroud\AzureFunctions\AzureFunctions\bin\Debug\net8.0\AzureFunctions.dll
  Sonar: (AzureFunctions.csproj) Project processed successfully
```